SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WorkerCoordinator/ManagedWorkerCoordinator.cs

```diff
@@ -212,8 +211,8 @@ private void WaitForTargetDeploymentReady(Connection connection)
         {
             while (true)
             {
-                var readyFlagOpt = connection.GetWorkerFlag(TargetDeploymentReadyWorkerFlag);
-                if (readyFlagOpt.HasValue && readyFlagOpt.Value.ToLower() == "true")
+                string readyFlag = connection.GetWorkerFlag(TargetDeploymentReadyWorkerFlag);
+                if (String.Compare(readyFlag, "true", true) == 0)
```

Comment:
seems dutty but 🤷

Reply:
Had to google "dutty", I assume you're using it here in the positive term - eg "Booya! That freestyle was dutty!"

Reply:
In the off-chance it was the negative term, I'm open to suggestions.

Reply:
> Booya! That freestyle was dutty!

precisely